README.md

```diff
@@ -92,6 +92,7 @@ Click the button below to start deployment.
    - **Event Hub Connection String** - Type the connection string for the existing Event Hub; leave empty if creating a new Event Hub.
    - **Event Hub Namespace** - Type the namespace for the existing Event Hub; leave empty if creating a new Event Hub.
    - **Event Hub Name** - Type the name of the existing Event Hub.
+   - **Event Hub Filter** - To check what type of filtering is supported click [here](#event-hub-filtering)
 
    **Note:** This value defaults to `insight-operational-logs`. This Event Hub is created automatically by Azure when a subscription [Log Profile is integrated with Event Hub through the Azure Monitor service](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/stream-monitoring-data-event-hubs#azure-subscription-monitoring-data).
```

Comment:
this note belongs to `Event Hub Name`. Please place it after `Event Hub Name` parameter.

Reply:
Done